src/app/settings/views/Configuration/Commissioning/Commissioning.test.tsx

```diff
@@ -48,7 +48,7 @@ describe("Commissioning", () => {
     state.config.loading = true;
     const store = mockStore(state);
 
-    const wrapper = mount(
+    render(
```

Comment:
One thing you could consider doing when migrating these tests is use the render utils `renderWithMockStore` or `renderWithBrowserRouter` which would simplify these tests a little.

This render would then become:
```typescript
renderWithMockStore(<Commissioning />, {
  state,
});
```

Reply:
Thanks for this! That's gonna help a lot in converting future tests. Only bit in this file I couldn't do is the third test, since that relies on the store to check actions, and so it needs to be defined beforehand so that the render is using the same store that actions are being fetched from.

Reply:
> Thanks for this! That's gonna help a lot in converting future tests. Only bit in this file I couldn't do is the third test, since that relies on the store to check actions, and so it needs to be defined beforehand so that the render is using the same store that actions are being fetched from.

At some stage we'll have to update those render methods to take the full store object for cases like this.

> 
> ETA: Turns out I also can't add this to the second test since it starts throwing errors from `useNavigate()`.

It's possible that one needs `renderWithBrowserRouter` so the routing is set up when the test runs.

> Gonna leave this for now rather than pushing

That's OK, this is just a bonus improvement.